src/Symfony/Bridge/PhpUnit/Tests/DnsMockTest.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Tests;
+
+use Symfony\Bridge\PhpUnit\DnsMock;
+
+require_once __DIR__.'/../DnsMock.php';
+
+class DnsMockTest extends \PHPUnit_Framework_TestCase
+{
+    protected function tearDown()
+    {
+        DnsMock::withMockedHosts(array());
+    }
+
+    public function testCheckdnsrr()
+    {
+        DnsMock::withMockedHosts(array('example.com' => array(array('type' => 'MX'))));
```

Comment:
The docblock states that the argument should be an array that is structured like what would be returned by `dns_get_record()`. Obviously, we don't need to do that here as the tests do pass anyway, but shouldn't we add the missing keys nonetheless for people who look at the tests to check how to exactly create their own mocks?